setup.cfg

```diff
@@ -38,4 +38,4 @@ testing =
 scraping = py.typed
 
 [flake8]
-max-line-length = 160
```

Comment:
how did you choose the number of lines in flake8?
Is there a recommendation? like this pep8?

Reply:
this is the default that we found in github actions template, when you click `Python application` 
<img width="901" alt="image" src="https://user-images.githubusercontent.com/15682787/182304742-6080f520-ac34-417e-b2b8-2dac05c41872.png">